tests/components/fujitsu_fglair/test_config_flow.py

```diff
@@ -105,3 +105,41 @@ async def test_form_exceptions(
         CONF_PASSWORD: TEST_PASSWORD,
         CONF_EUROPE: False,
     }
+
+
+async def test_reauth_success(
```

Comment:
Please also test the exceptions, even though the lines are covered by the original tests, we want to completely test all the paths